src/pkg/packaging/dir.proj

```diff
@@ -160,18 +160,19 @@
 
   <Target Name="GenerateNugetPackages" DependsOnTargets="InitPackage">
     <ItemGroup>
-      <PackageProjects Include="$(ProjectDir)src\managed\Microsoft.DotNet.PlatformAbstractions\project.json" />
-      <PackageProjects Include="$(ProjectDir)src\managed\Microsoft.Extensions.DependencyModel\project.json" />
+      <PackageProjects Condition="'$(OS)' == 'Windows'" Include="$(ProjectDir)src\managed\Microsoft.DotNet.PlatformAbstractions\Microsoft.DotNet.PlatformAbstractions.csproj" />
```

Comment:
We should see if we can fix this.  It works in the CLI repo to "build" for netstandard, and then later come back and say `dotnet pack --no-build` on non-Windows.  I'm assuming there is more going on here, since this works in other repos.